packages/form-core/src/choice-group/ChoiceGroupMixin.js

```diff
@@ -157,7 +157,11 @@ export const ChoiceGroupMixin = dedupeMixin(
       _getFromAllFormElements(property, filterCondition = () => true) {
         // For modelValue and serializedValue, an exception should be made,
         // The reset can be requested from children
-        if (property === 'modelValue' || property === 'serializedValue') {
```

Comment:
nitpick: comment should also mention formattedValue

Reply:
updated